pyatv/helpers.py

```diff
@@ -69,6 +70,11 @@ def get_unique_id(  # pylint: disable=too-many-return-statements
         return properties.get("UniqueIdentifier")
     if service_type == AIRPLAY_SERVICE:
         return properties.get("deviceid")
+    if service_type == COMPANION_SERVICE:
+        # Apple TV devices on tvOS 16 (maybe earlier) have a static rpMRtID
+        # identifier. For other devices use rpBA which periodically changes but is
+        # still useful.
+        return properties.get("rpmrtid") or properties.get("rpba")
```

Comment:
the identifer here must be unique, so it is ok to keep `rpmrtid` but not `rpba`.

Reply:
Since only Apple TVs report `rpmrtid` (which I assume is an identifier for the remote app), I added `rpba` to be able to more easily run scripts against the proxy connected to a HomePod (otherwise it has no identifier). The `rpba` seams to be unique although it isn't permanent. I'll remove it from the PR though.

Reply:
Done.